ratslam_view_template:
  ros__parameters:
    image_crop_x_min: 40
    image_crop_x_max: 600
    image_crop_y_min: 150
    image_crop_y_max: 300
    vt_match_threshold: 0.085
    template_x_size: 60
    template_y_size: 10
    vt_shift_match: 5
    vt_step_match: 1
    vt_normalisation: 0.4
    # draw
    enable: true
    vt_window_width: 640
    vt_window_height: 900

ratslam_pose_cells:
  ros__parameters:
    pc_cell_x_size: 2.0
    pc_dim_xy: 30
    pc_vt_inject_energy: 0.15
    exp_delta_pc_threshold: 1.0
    vt_active_decay: 1.0
    pc_vt_restore: 0.04
    # draw
    enable: true

ratslam_experience_map:
  ros__parameters:
    exp_loops: 20
    # draw
    enable: true

ratslam_visual_odometry:
  ros__parameters:
    vtrans_image_x_min: 195
    vtrans_image_x_max: 475
    vtrans_image_y_min: 270
    vtrans_image_y_max: 430

    vrot_image_x_min: 195
    vrot_image_x_max: 475
    vrot_image_y_min: 75
    vrot_image_y_max: 240

    camera_fov_deg: 53.0
    camera_hz: 10.0
    vtrans_scaling: 1000.0
    vtrans_max: 20.0